src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -24,20 +27,31 @@ const formConfig = {
   title: 'Apply for education benefits',
   subTitle: 'Form 22-1990',
   chapters: {
-    veteranInformation: {
-      title: 'Veteran Information',
+    applicantInformation: {
+      title: 'Applicant Information',
       pages: {
-        veteranInformation: {
-          title: 'Veteran information',
-          path: 'veteran-information',
+        applicantInformation: _.merge(applicantInformation(fullSchema1990, {
+          isVeteran: true,
+          fields: [
+            'veteranFullName',
+            'veteranSocialSecurityNumber',
+            'veteranDateOfBirth',
+            'gender'
+          ],
+        }), {
           uiSchema: {
-          },
-          schema: {
-            type: 'object',
-            properties: {
+            veteranDateOfBirth: {
+              'ui:validations': [
+                (errors, dob) => {
+                  // TODO: Use a constant instead of 'YYYY-MM-DD'
+                  if (moment(dob, 'YYYY-MM-DD').isAfter(moment().endOf('day').subtract(17, 'years'))) {
```

Comment:
Yeah, I mean it will return `false` for invalid dates (I tested), but when the date doesn't fit the format supplied, that's when some funny things happen.

If the `dob` is `12-12-1991`, funny things happen: http://jsbin.com/balenepimo/2/edit?js,console
This will come out to `Wed Dec 19 2012 00:00:00 GMT-0800` which...is wrong.

I'm open to suggestions, but I'm not sure how else to do that. I mean, I could do the parsing myself, but that seems a little heavy-handed, doesn't it?